collect_app/google-services.json

```diff
@@ -1,43 +1,2 @@
-{
-  "project_info": {
-    "project_number": "895206029843",
-    "project_id": "opendatakit-collect-forks"
-  },
-  "client": [
-    {
-      "client_info": {
-        "mobilesdk_app_id": "1:895206029843:android:5e1d7c3c296329e8",
-        "android_client_info": {
-          "package_name": "org.odk.collect.android"
-        }
-      },
-      "oauth_client": [
-        {
-          "client_id": "895206029843-btufcdli1qptfd4sjrq5f0vdacsg68fj.apps.googleusercontent.com",
-          "client_type": 3
-        }
-      ],
-      "api_key": [
-        {
-          "current_key": "AIzaSyAD_MRym4ELgrqCJ4IGkPAdkfPXYX1WDeI"
-        }
-      ],
-      "services": {
-        "analytics_service": {
-          "status": 2,
-          "analytics_property": {
-            "tracking_id": "UA-111083836-1"
-          }
-        },
-        "appinvite_service": {
-          "status": 1,
-          "other_platform_oauth_client": []
-        },
-        "ads_service": {
-          "status": 1
-        }
-      }
-    }
-  ],
-  "configuration_version": "1"
-}
\ No newline at end of file
+The content of this file should be private to ensure it is used only for oficiall releases. It should be filled
```

Comment:
This is actually the forks file. We don't have a Firebase project for forks yet so it can be left as-is. As described [in the readme](https://github.com/opendatakit/collect#creating-signed-releases-for-google-play-store), `collect_app/src/odkCollectRelease/google-services.json` is where the official release file goes.

Reply:
but such a folder (odkCollectRelease) doesn't even exist... does that mean you create it and put the config file there? If so the one I wanted to remove here is redundant because forks have it's own in release dir. I thought that one is used for our official releases.

Reply:
Yes, that folder gets created. I believe that the one you are removing/replacing here will act as a fallback for someone who introduces a new variant for some reason. That's why it's identical to the forks one.

Reply:
Aha ok makes sense.